src/devices/localization2DServer/localization2DServer.h

```diff
@@ -0,0 +1,67 @@
+/*
+* Copyright (C)2017 ICub Facility - Istituto Italiano di Tecnologia
+* Author: Marco Randazzo
+* email:  marco.randazzo@iit.it
+* website: www.robotcub.org
+* Permission is granted to copy, distribute, and/or modify this program
+* under the terms of the GNU General Public License, version 2 or any
+* later version published by the Free Software Foundation.
+*
+* A copy of the license can be found at
+* http://www.robotcub.org/icub/license/gpl.txt
+*
+* This program is distributed in the hope that it will be useful, but
+* WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+* Public License for more details
+*/
+
+#include <yarp/os/Network.h>
+#include <yarp/os/RFModule.h>
+#include <yarp/os/Time.h>
+#include <yarp/os/Port.h>
+#include <yarp/os/BufferedPort.h>
+#include <yarp/os/PeriodicThread.h>
+#include <yarp/dev/PolyDriver.h>
+#include <yarp/dev/Wrapper.h>
+#include <yarp/dev/ControlBoardInterfaces.h>
+#include <yarp/dev/ILocalization2D.h>
+#include <math.h>
+
+#ifndef LOC_SERVER_H
+#define LOC_SERVER_H
+
+#define DEFAULT_THREAD_PERIOD 0.02 //s
+
+class localization2DServer : public yarp::dev::DeviceDriver,
```

Comment:
It shouldn't be under the namespace `yarp::dev`?
Same thing for the `navigation2DServer`